capstone-app/src/app/modules/photos/components/photos-section/photos-section.component.html

```diff
@@ -1 +1,21 @@
-<app-image-container></app-image-container>
\ No newline at end of file
+<div class="flex-container">
+    <form class="count">
+      <mat-form-field>
+        <mat-label>Number</mat-label>
+        <input
+          matInput
+          type="number"
+          placeholder="Ex. 10"
+          [formControl]="limitControl"
+          min="1"l
```

Comment:
seems to be a trailing 'l' character at the end of the line. Is this intentional?